### PR TITLE
Get reviews

### PR DESCRIPTION
**Pull request by ZacharyAb**

### CODE REVIEW

File: server/__test__/username.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import UsernameAndDate from '../client/components/username';
+
+describe('<username />', () => {
+  it('should properly format the date', () => {
+    expect(UsernameAndDate.dateFormatter('2019-04-06')).toEqual('Apr 06, 2019');
+  });
+});
```
Comment (naszzz) [inline server/__test__/username.test.js]: this is cool. didn't know about this dateFormatter method.

File: server/client/components/app.jsx [hunk 2]
```diff
@@ -2,12 +2,18 @@ import React from 'react';
...
-const App = () => (
-  <div>
-    <ReviewBox />
+const App = () => {
+  return (
+    <div>
+    <ReviewBox id={id} />
+    <ReviewBox id={25} />
+    <ReviewBox id={26} />
+    <ReviewBox id={27} />
```
Comment (naszzz) [inline server/client/components/app.jsx]: are these hardcoded id values temporary? in future will they be values incremented from id ?

File: server/client/components/avatar.jsx [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
-const axios = require('axios');
 
-const Avatar = styled.div`
+const Avatar = styled.img`
   border-radius: 1.3em;
   height: 50px;
   width: 50px;
+  grid-column-start: col-one;
+  grid-row-start: row-one;
+  grid-row-end: row-two;
```
Comment (naszzz) [inline server/client/components/avatar.jsx]: cool. so gird can be used within styled components and without importing anything extra. good to know.

File: server/client/components/username.jsx [hunk 4]
```diff
@@ -4,26 +4,38 @@ import styled from 'styled-components';
...
 
+const dateFormatter = (date) => {
+  const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+  let newDate = '';
+  let arrOfDates = date.split('-');
+  arrOfDates[1] = months[arrOfDates[1] - 1];
+  newDate += arrOfDates[1] + ' ' + arrOfDates[2] + ', ' + arrOfDates[0];
+  return newDate;
+};
+
```
Comment (naszzz) [inline server/client/components/username.jsx]: lgtm

File: server/client/components/username.jsx [hunk 5]
```diff
@@ -4,26 +4,38 @@ import styled from 'styled-components';
...
+  grid-row-start: row-one;
+  order: one;
 `;
 
 const Date = styled.div`
-  color: blue;
+  grid-column-start: col-two;
+  grid-row-start: row-one;
+  order: two;
+  padding-left: 10px;
```
Comment (naszzz) [inline server/client/components/username.jsx]: grid looks super useful.

File: server/__test__/username.test.js (hunk 1 above)
Comment (ZacharyAb) [reply server/__test__/username.test.js]: I created it in another file.

File: server/client/components/app.jsx (hunk 2 above)
Comment (ZacharyAb) [reply server/client/components/app.jsx]: Yes they are temporary just to see what the render would look like for styling.

File: server/client/components/avatar.jsx (hunk 3 above)
Comment (ZacharyAb) [reply server/client/components/avatar.jsx]: Yeah! Grid is built in with CSS.

**Pull request merged**